lib/sample_uploader/utils/importer.py

```diff
@@ -495,6 +496,12 @@ def import_samples_from_file(
             e.sample_name = err_row_sample_names[e.row]
         if e.row==None and e.sample_name!=None and e.sample_name in err_sample_name_indices:
             e.row = err_sample_name_indices[e.sample_name]
+        if e.subkey:
+            for group in column_groups:
+                if e.subkey in group and e.key == group["value"]:
+                    e.column = err_key_indices[group[e.subkey]]
+                    break
+        e.column_name = err_col_keys[e.column]
```

Comment:
Is `e.column` sometimes None?

Reply:
hmm, not in existing usecases but just in case I'll use .get :)